curve25519-dalek/src/backend/serial/risc0/constants.rs

```diff
@@ -0,0 +1,183 @@
+use super::field::FieldElementR0;
+use super::scalar::ScalarR0;
+use crate::edwards::EdwardsPoint;
+
+use elliptic_curve::bigint::U256;
+
```

Comment:
These constants are taken from the u64/u32 existing implementation (https://github.com/risc0/curve25519-dalek/blob/main/curve25519-dalek/src/backend/serial/u64/constants.rs)